pallets/automation-time/src/lib.rs

```diff
@@ -360,6 +346,23 @@ pub mod pallet {
 			who: AccountOf<T>,
 			task_id: TaskIdV2,
 		},
+		TaskTriggered {
+			who: AccountOf<T>,
+			task_id: TaskIdV2,
+			condition: BTreeMap<String, String>,
+			encoded_call: Vec<u8>,
+			/// List of errors causing task abortion.
+			abort_errors: Vec<String>,
```

Comment:
Good call,

TaskTriggered doesn’t need this array cuz it’s too noisy.

Neither does TaskNotRescheduled need it, because it already contains the actual error that caused the NotRescheduled event. We can always look up the task creation event to verify that the causing error is in the abort_errors array.

TaskRescheduleFailed doesn’t need this because that error is most likely to be caused by block full and not relate to abort_errors.

Reply:
Fixed